script/tool/lib/src/common.dart

```diff
@@ -584,7 +596,7 @@ class PubVersionFinder {
 
   /// Get the package version on pub.
   Future<PubVersionFinderResponse> getPackageVersion(
-      {@required String package}) async {
+      {required String? package}) async {
```

Comment:
can be `required String package` and remove the below assert for != null

Reply:
Done.